### ISSUE TITLE
Update Production Methods for pre-1836 techs

### ISSUE THREAD

**Issue by trym-b** [AndHope/Basileia-Romaion#52]
When Spheres of Influence releases in 2023-05-XX, production methods should be overhauled for the artisan PMs.

The eras that should be overhauled are mainly the era 1 techs, especially production techs.

**Comment by AndHope** [comment]
Yeah, if you still have interest in that go for it would be really cool to work with you again. salute
